Matrix.py

```diff
@@ -0,0 +1,123 @@
+class Matrix:
+    def __init__(self, x):
+        if type(x) is tuple:
+            for tup in x:
+                if len(tup) == len(x) and type(tup) is tuple:
+                    self.x = x
+                else:
+                    self.x = None
+                    print("invalid Matrix")
+        else:
+            self.x = None
+            print("invalid matrix")
+
+    def __iter__(self):
+        return iter(self.x)
+
+    def __getitem__(self, item):
+        return self.x[item]
+
+    def __repr__(self):
+        string = ""
+        for i in range(len(self.x)):
+            string += (str(self.x[i]) + "\n")
+        return string
+
+    def __str__(self):
+        return "Matrix:" + str(self.x)
+
+    def get_tuple(self):
+        return self.x
+
+    @staticmethod
+    def unity(unit):
+        ls = []
+        for i in range(unit):
+            ls.append([0] * unit)
+            ls[i][i] = 1
+        unitup = [tuple(lt) for lt in ls]
```

Comment:
rename unitup

Reply:
why? its a tuple of the unit matrix (unit-tuple)